presto-mongodb/src/test/java/io/prestosql/plugin/mongodb/TestMongoDistributedQueries.java

```diff
@@ -25,23 +24,16 @@
 public class TestMongoDistributedQueries
         extends AbstractTestQueries
 {
-    private MongoQueryRunner mongoQueryRunner;
+    private static final MongoQueryRunner QUERY_RUNNER = createMongoQueryRunner(TpchTable.getTables());
```

Comment:
I would like to avoid to have testing presto server created during static variables initialization.

Reply:
https://github.com/prestosql/presto/pull/2038